pid_node:
  ros__parameters:
    P_gain: [0.0, 0.0, 0.0, 0.2, 0.2, 0.2]
    I_gain: [0.0, 0.0, 0.0, 0.2, 0.2, 0.2]
    D_gain: [0.0, 0.0, 0.0, 0.3, 0.3, 0.3]
    I_lim: [0.0, 0.0, 0.0, 1.0, 1.0, 1.0]
    Out_lim: [0.0, 0.0, 0.0, 5.0, 5.0, 5.0]
    antiwindup: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    pid_enable: [false, false, false, true, true, true]
    cutoff_frequency: 5.
    angle_wrap_compensation: [false, false, false, true, true, true]
    rate: 30

